### ISSUE TITLE
feat(a11y): Chat-input should have a describable aria-label instead of placeholder

### ISSUE THREAD

**Issue by cguhl-adesso** [Rudo-hayate/open-webui-a11y#21]
### Check Existing Issues

- [x] I have searched for all existing **open AND closed** issues and discussions for similar requests. I have found none that is comparable to my request.

### Verify Feature Scope

- [x] I have read through and understood the scope definition for feature requests in the Issues section. I believe my feature request meets the definition and belongs in the Issues section instead of the Discussions.

### Problem Description

The chat-input has only a describable text as placeholder. For users with disabilities it´s not obvious what the component is doing because placeholders are not a valid descriptions because they are not fixed and disappear when user is entering some text.

<img width="1097" height="466" alt="Image" src="https://github.com/user-attachments/assets/fcc30155-e807-45b7-bc85-7735077950c2" />

### Desired Solution you'd like

- add aria-label to the chat-input-component instead of the placeholder to describe it´s function
